lib/signaling/v3/remoteparticipant.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const RemoteParticipantV2 = require('../v2/remoteparticipant');
+const RemoteTrackPublicationV3 = require('./remotetrackpublication');
+
+/**
+ * @extends RemoteParticipantV2
+ */
+class RemoteParticipantV3 extends RemoteParticipantV2 {
+  /**
+   * Construct a {@link RemoteParticipantV2}.
+   * @param {object} participantState
+   * @param {function(Track.SID): Promise<MediaTrackReceiver>} getPendingTrackReceiver
+   * @param {function(Track.SID): boolean} getInitialTrackSwitchOffState
+   * @param {function(Track.SID, Track.Priority): boolean} setPriority
+   * @param {function(Track.SID, ClientRenderHint): Promise<void>} setRenderHint
+   * @param {function(Track.SID): void} clearTrackHint
+   * @param {object} [options]
+   */
+  constructor(
+    participantState,
+    getPendingTrackReceiver,
+    getInitialTrackSwitchOffState,
+    setPriority,
+    setRenderHint,
+    clearTrackHint,
+    options
+  ) {
+    options = Object.assign({
+      RemoteTrackPublicationSignaling: RemoteTrackPublicationV3,
+      otherProperties: {
```

Comment:
curious why otherProperties is a separate object.

Reply:
This should be resolved in the next commit.